pmm-app/src/pmm-qan/panel/provider/provider.tsx

```diff
@@ -215,8 +216,13 @@ export const UrlParametersProvider = (props) => {
     if (newTo === 'now') {
       setToTimeMomentValue(timeRange.to.subtract(1, 'minute')
         .format('YYYY-MM-DDTHH:mm:ssZ'));
-      setFromTimeMomentValue(timeRange.from.subtract(1, 'minute')
-        .format('YYYY-MM-DDTHH:mm:ssZ'));
+
+      if (moment.isMoment(timeRange.raw.from)) {
```

Comment:
What was exactly the issue?

Reply:
https://jira.percona.com/browse/PMM-9987 
as per above ticket minus 1 minute was only required when the 'From' and 'To' time is relative (ie ranges like now-6h, now-30m etc.), Hence the special case is required when the 'From' time is absolute and To time is relative 'now'. Previous logic was doing minus 1 minute from the absolute 'From' time which was causing infinite loop.